lib/two_factor_authentication/models/two_factor_authenticatable.rb

```diff
@@ -87,6 +103,19 @@ def generate_totp_secret
           ROTP::Base32.random_base32
         end
 
+        def create_backup_codes(options = {})
+          count = options[:count] || self.class.backup_code_count || 10
+          digits = options[:length] || self.class.backup_code_length || 8
+          backup_codes = (1..count).map do
+            random_base10(digits)
```

Comment:
Any reason to use numbers only for codes as opposed to alphanumeric characters, like the devise-two-factor gem? GitHub and Apple also use alphanumeric codes.